views/index.erb

```diff
@@ -44,16 +162,32 @@
   </div>
 <% end %>
 
-
+	
 <!-- Add your Javascript to this script tag -->
 <script language="javascript">
   console.log("I am Javascript!");
-
-  var profile_values = document.getElementsByClassName('profile-value');
-  var total_words = 0;
-
-  // TODO - Count the number of words in the profile.
-  // hint - look at what the innerHTML property provides.
-
-  document.getElementById('word-count').innerHTML = total_words;
+	
+//count all words in the 4 textareas i tried to do it by Class instead of ID but it did not work
+	
+ function countBoxWords(to){
+	 	var box1words = document.getElementById('txt1').value.split(' ').length-1;
+		var box2words = document.getElementById('txt2').value.split(' ').length-1;
+		var box3words = document.getElementById('txt3').value.split(' ').length-1;
+		var box4words = document.getElementById('txt4').value.split(' ').length-1;
```

Comment:
Could you think of a way of repeating this code again and again without repeating yourself? How might a loop help?